src/php/comments.php

```diff
@@ -13,12 +13,3 @@
 $context['should_show_avatars'] = get_option( 'show_avatars' );
 
 Timber\Timber::render( 'partials/comments.twig', $context );
-
-/*
- * If the current post is protected by a password and
- * the visitor has not yet entered the password,
- * return early without loading the comments.
- */
-if ( post_password_required() ) {
```

Comment:
This was already here on an existing comments.php template I found, but it doesn't seem like we need it.